core/src/components/picker-column/picker-column.tsx

```diff
@@ -74,21 +75,27 @@ export class PickerColumnCmp implements ComponentInterface {
       onEnd: (ev) => this.onEnd(ev),
     });
     this.gesture.enable();
+    // Options have not been initialized yet
+    // Animation must be disabled through the `noAnimate` flag
+    // Otherwise, the options will will render
+    // at the top of the column and transition down
     this.tmrId = setTimeout(() => {
-      this.noAnimate = false;
```

Comment:
Why was this removed?

Reply:
I ended up moving it to the `update()` but I'll be reverting the `noAnimate` back and using another flag as requested.